app/services/language_indicator_service.js

```diff
@@ -24,4 +24,11 @@ const saveLanguageIndicator = (scorecardUUID, indicators) => {
   });
 }
 
-export {saveLanguageIndicator};
\ No newline at end of file
+const getLanguageIndicator = (scorecardUuid, indicatorId, type) => {
+  const scorecardPreference = realm.objects('ScorecardPreference').filtered(`scorecard_uuid == '${scorecardUuid}'`)[0];
+  let languageCode = type === 'audio' ? scorecardPreference.audio_language_code : scorecardPreference.text_language_code;
+  const languageIndicator = realm.objects('LanguageIndicator').filtered(`indicator_id == '${indicatorId}' AND language_code == '${languageCode}'`)[0];
+  return languageIndicator != undefined ? languageIndicator : '';
```

Comment:
@limkimsan why don't we return undefined instead of an empty string('')

Reply:
Bong, I have updated the code to return undefined. Thanks bong.